src/tests/integration-test

```diff
@@ -246,7 +251,8 @@ class UDisksTestCase(unittest.TestCase):
         If cd is True, return the CD device, otherwise the hard disk device.
         """
         block = klass.udisks_block(partition, cd)
-        return klass.client.get_object(block.get_object_path()).get_filesystem()
+        return klass.client.get_object(
+            block.get_object_path()).get_filesystem()
```

Comment:
@vpodzime I was just trying to adhere to the pep standard with my goal of being able to run the pep8 tool and pylist tools clean.  At the moment we don't appear to be adhering to anything really.  If the code was passing clean with these tools, it would be easy to check for future violations when adding/changing new code.  At the moment things are so non-conforming that anything new will just be lost in the noise. 

As for it just being *tests*, I think we should be striving to make the tests be as good as the code it's testing in all ways, including code formatting etc.